paddle/fluid/operators/lookup_table_op.h

```diff
@@ -102,7 +102,8 @@ class LookupTableKernel : public framework::OpKernel<T> {
             auto id_index = table_t.GetIndexFromId(ids[i]);
 
             if (id_index != -1) {
-              if (input_data_type == framework::proto::VarType::INT8) {
+              if (input_data_type == framework::proto::VarType::INT8 ||
+                  input_data_type == framework::proto::VarType::BF16) {
```

Comment:
I wonder why do you change `lookup_table_op` but not `lookup_table_v2_op`?

Reply:
That's because in the model "word2vec" there is `lookup_table` op used. Moreover [`paddle.fluid.layers.embedding`](https://github.com/PaddlePaddle/Paddle/blob/d3ec88ad2b22515038618df4404955711650f92d/python/paddle/fluid/layers/nn.py#L510-L520) Py API creates `lookup_table` op.